packages/shared/src/utils/object/object.ts

```diff
@@ -1,18 +1,19 @@
+/* eslint-disable */
```

Comment:
The reason for `any` types in this component is we are passing different types of properties to these functions which are not unique, so I had to add `any` types!